tests/shell/environment/env.rs

```diff
@@ -86,6 +86,22 @@ fn env_assignment() {
     assert_eq!(actual.out, "barbaz");
 }
 
+#[test]
+fn env_assignment_with_if() {
+    let actual = nu!(cwd: ".", r#"
```

Comment:
i do not think we need these two `cwd` here :yum:

Reply:
Yeah I think so, thanks for catching this!